src/univariate/continuous/chi.jl

```diff
@@ -80,9 +80,11 @@ end
 
 #### Evaluation
 
-logpdf(d::Chi, x::Real) = (ν = d.ν;
+function logpdf(d::Chi, x::Real)
+    x < 0 && return 0.0
```

Comment:
This is type unstable, better to compute the result as much as possible (as done for other distributions).
```suggestion
```

Reply:
BTW one should also not return `0` but `-Inf` (of the correct type).

Reply:
Ah sorry, was thinking of `pdf`.